wiki/Registration/zh.md

```diff
@@ -0,0 +1,21 @@
+# 注册
+
+*警告: 任何时候拥有一个以上的osu！账户都违反 [osu! 规则](/wiki/Rules)!*
+
+1. 在 [安装osu!](/wiki/Installation) 并运行之后，osu!将提示你登入或之策。 假设你现在没有账户，请点击 `创建账户`。
+
+   ![游戏内注册界面](img/ingame-registration.jpg "注册表格")
+
+2. 填写表格中的字段。
+   - 用户名最多只能有15个字符，用户名只能由字母 (`a-z`, `A-Z`, `0-9`)，下划线 (`_`)，方括号(`[` 和 `]`)，破折号(`-`)，和空格(` `)组成。
+   - 对于电子邮件地址字段，请使用你的邮箱。这将在你需要重置密码时给你发送验证码。所以不要丢失你的邮箱。
+   - 对于密码字段，请使用不少于8个字符的密码。
+3. 一旦完成注册，点击 `1. Create my account`。你将会自动登录。
```

Comment:
`1. Create my account` should be translated into Chinese. (It can be found in the osu! client.)

Reply:
When I changed osu! to Chinese.This button is in English
[This picture](https://s2.ax1x.com/2019/12/20/QLqwtO.png)
so I think I should be Create my account

Reply:
yeah for now the registration page can't be translated. https://github.com/ppy/osu-stable-issues/issues/228

so leave this in English